mindmeld/components/nlp.py

```diff
@@ -661,17 +673,15 @@ def inspect(self, markup, domain=None, intent=None, dynamic_resource=None):
             domain_inspection = self.domain_classifier.inspect(
                 query, domain=domain, dynamic_resource=dynamic_resource
             )
```

Comment:
The doc strings for domain_classifier.inspect(), intent_classifier.inspect() and DomainProcessor.inspect() also say they return data frames.